### PR TITLE
Camera limiter

### PR DESCRIPTION
**Pull request by TalPat**
Camera that has a behavior similar to that of the original Bomberman

### CODE REVIEW

File: renderer/src/Renderer.cpp [hunk 1]
```diff
@@ -354,7 +354,19 @@ void Renderer::render(sf::RenderWindow &window, const GameState &state)
 	_clock.restart();
 
 	sf::Vector2f playerPosition(state.player.position());
-	playerPosition -= sf::Vector2f(0.5, 0.5);
+	playerPosition -= sf::Vector2f(0.5, 0.5);	int camoffset = 7;
```
Comment (GJSteyn) [inline renderer/src/Renderer.cpp]: Should probably be on a new line

File: renderer/src/Renderer.cpp [hunk 2]
```diff
@@ -354,7 +354,19 @@ void Renderer::render(sf::RenderWindow &window, const GameState &state)
...
+	if (playerPosition.x < camoffset) {
```
Comment (GJSteyn) [inline renderer/src/Renderer.cpp]: Couple java brackets

File: renderer/src/Renderer.cpp [hunk 3]
```diff
@@ -354,7 +354,19 @@ void Renderer::render(sf::RenderWindow &window, const GameState &state)
...
+	}
+	if (playerPosition.y < camoffset) {
+		playerPosition.y = camoffset;
+	}
+	if (playerPosition.x > state.map.size().x - camoffset - 1) {
+		playerPosition.x = state.map.size().x - camoffset - 1;
+	}
+	if (playerPosition.y > state.map.size().y - camoffset - 1) {
+		playerPosition.y = state.map.size().y - camoffset - 1;
+	}
```
Comment (GJSteyn) [inline renderer/src/Renderer.cpp:374]: Is there any reason that some of these conditionals can't be grouped into 'else if' blocks. Just think that some of them could exclude the possibility of others being true?

**Review by dpoulter-WTC — COMMENTED** [review]
What is the file renderer/res/sprites/computorv1.txt ?

**Comment by TalPat** [comment]
Accidentally left an unrelated text file in sprites. Removed now

**Review by GJSteyn — CHANGES_REQUESTED** [review]

File: renderer/src/Renderer.cpp (hunk 3 above)
Comment (TalPat) [reply renderer/src/Renderer.cpp:374]: exclusive conditions made into if-else pairs

**Review by GJSteyn — APPROVED** [review]

**Pull request merged**